conda-forge.yml

```diff
@@ -7,3 +7,5 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+azure:
+  store_build_artifacts: True
```

Comment:
This change was for debugging. I should remove it and rerender after everyone is done reviewing.

Reply:
is it time now?

Reply:
Nope. I'm going to shrink this PR using the new conda-build features.

Reply:
ok curious to see!